models/intel/noise-suppression-poconetlike-0001/noise-suppression-poconetlike-0001.md

```diff
@@ -0,0 +1,262 @@
+# noise-suppression-poconetlike-0001
```

Comment:
please note, starting from 2021.4 we have changed names of model descriptions files to simple README.md